### ISSUE TITLE
Bug with unit test when rendering website automatically

### ISSUE THREAD

**Issue by fhoces** [BITSS-OPA/opa-deworming#103]
Whenever we render the DD into a `index.html` file using the YAML command: 

```
  (function(input_file, encoding) {
    rmarkdown::render(input_file, encoding=encoding, output_file=file.path("..", 'index.html'))
  })
```

A unit test in the DD fails: 
<img width="974" alt="Screen Shot 2021-02-01 at 3 51 39 PM" src="https://user-images.githubusercontent.com/2438124/106532637-63b32b80-64a5-11eb-9d88-6af753243906.png">

**Comment by fhoces** [comment]
replace this with 4 smaller tasks starting in line 94

**Comment by keanulim123** [comment]
Completed:

Task 1) Added a note above the rmarkdown:render line
Task 2) Setting purl to FALSE omits chunks from the all_analysis.R file. I've set this to false for the packages code chunk.
Task 4) Created a csv file called ea3_pe that saves the previous version of ea3_pe. This is assigned to main_pe_so when knitting

Uncompleted:

Task 3) I'm not sure how to save the plot without moving the code to create the plot to the top of the Rmd file. Currently, the plot is generated manually in line 343 from the images directory ('main_pe.png') which requires the user to have knit the Rmd at least once (or for main_pe.png to exist somehow i.e. by forking the repo). I think this is fine and I've mirrored this process to complete Task 4.
